conbench/templates/app.html

```diff
@@ -91,21 +83,20 @@
                 <div class="modal-content">
                   <div class="modal-header">
                     <h4 class="modal-title">
-                      <i class="bi bi-trash3-fill"></i>
+                      <i class="bi bi-trash3-fill"></i> garbage or not? maybe it's art?
```

Comment:
Haha, what does this mean?

<img width="481" alt="image" src="https://github.com/conbench/conbench/assets/16600275/36cea664-8fdf-420e-8046-eaef0bc7f565">

Reply:
:D In German we have the saying 

>  Ist das Kunst oder kann das Weg?

In English this is like "trash or treasure?" and here I took a freestyle approach. Let me know if that's too stupid / not good :).

Reply:
No I like the whimsy. Software shouldn't take itself too seriously :)